### PR TITLE
Add vm status

### PR DESCRIPTION
**Pull request by rnetser**
##### Short description:
Add VM Status.STARTING and move Status.RUNNING to resource

### CODE REVIEW

**Comment by sarahbx** [comment]
👍 lgtm

File: ocp_resources/virtual_machine.py [hunk 1]
```diff
@@ -26,6 +26,14 @@ class RunStrategy:
         ALWAYS = "Always"
         RERUNONFAILURE = "RerunOnFailure"
 
+    class Status(NamespacedResource.Status):
```
Comment (myakove) [inline ocp_resources/virtual_machine.py:29]: All the statuses should be under resource

Comment (rnetser) [reply ocp_resources/virtual_machine.py:29]: If you have a status that's specific for a card I do not think it should go under resource

**Pull request merged**